### PR TITLE
GeneralButton from Component to CSS class

### PR DESCRIPTION
**Pull request by yageljaoui**
switched from GeneralButton Component to generalButton CSS class.

### CODE REVIEW

**Review by odeyah — APPROVED** [review]
checked and all looks good

File: src/GeneralComponents/GeneralWindow.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default function GeneralWindow({ children, unmountGeneralWindow, rightSid
 
 	return (
 		<div className={composedClassName}>
-			<GeneralCloseButton className='generalWindowCloseButton' unmountWindow={unmountGeneralWindow} />
+			<GeneralCloseButton className='generalButton closeButton' unmountWindow={unmountGeneralWindow} />
```
Comment (elShailee) [inline src/GeneralComponents/GeneralWindow.jsx:11]: there's no need for className here at all.

File: src/Screens/Gameplay/Components/Coins/CoinsTab.jsx [hunk 2]
```diff
@@ -18,9 +17,9 @@ export default function CoinsTab() {
 			{texts.gameplay.coins.coinsTab.coinsAmountPt1}
 			XX
 			{texts.gameplay.coins.coinsTab.coinsAmountPt2}
-			<GeneralButton id='getMoreCoinsButton' onButtonClick={getMoreButtonClickHandler}>
+			<div className='generalButton' id='getMoreCoinsButton' onClick={getMoreButtonClickHandler}>
```
Comment (elShailee) [inline src/Screens/Gameplay/Components/Coins/CoinsTab.jsx:20]: this button wasn't uppercased, please keep it as it were.
can be done by adding `text-transform: none; border: 0px;` to it's id.

File: src/Screens/Gameplay/Components/Energy/EnergyTab.jsx [hunk 3]
```diff
@@ -36,9 +35,9 @@ export default function EnergyTab() {
 				{texts.gameplay.energy.energyTab.perHour}
 			</div>
 
-			<GeneralButton id='getMoreEnergyButton' onButtonClick={() => getMoreButtonClickHandler()}>
+			<div className='generalButton' id='getMoreEnergyButton' onClick={() => getMoreButtonClickHandler()}>
```
Comment (elShailee) [inline src/Screens/Gameplay/Components/Energy/EnergyTab.jsx:38]: this button wasn't uppercased, please keep it as it were.
can be done by adding `text-transform: none; border: 0px;` to it's id.

**Pull request merged**